src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/convert_color_nv12.cpp

```diff
@@ -4,55 +4,91 @@
 
 #include <vector>
 
-#include "single_layer_tests/convert_color_nv12.hpp"
+#include "single_op_tests/convert_color_nv12.hpp"
 #include "common_test_utils/test_constants.hpp"
 
-using namespace LayerTestsDefinitions;
-
 namespace {
+using ov::test::ConvertColorNV12LayerTest;
+
+const std::vector<ov::Shape> in_shapes = {
+    {1, 20, 10, 1}
```

Comment:
are shapes same as before your changes? If no, I suggest to port test as is, and changes test parameters in a separate PR